metricbeat/docker-compose.yml

```diff
@@ -15,32 +15,56 @@ services:
 
   # Modules
   aerospike:
-    build: ./module/aerospike/_meta
+    image: docker.elastic.co/observability-ci/beats-integration-aerospike:${AEROSPIKE_VERSION:-3.9.0}-1
+    build:
+      context: ./module/aerospike/_meta
+      args:
+        AEROSPIKE_VERSION: ${AEROSPIKE_VERSION:-3.9.0}
```

Comment:
A drawback of this implementation is that default versions are duplicated in each use of the variables in the docker compose file, but they are quite together in the file so it should be difficult to forget to modify one when modifying the other.

Reply:
Let's try to improve this in the near future. As far as I think, in case of error; the output can be very  tricky to understand and lead to long periods of debugging if you don't take a calm, closer look at this.

I understand it's not something that is gonna happen commonly but it's also not the first thing that we have to be careful with.

Reply:
Sure, this will need to be done for the next steps so build and cache/push of all images can be automated, without depending on running all the tests.

Regarding the debugging required when mistakes are done here, I agree that any improvement there will be welcome, this is something that can be done in parallel to these changes to tag and cache images.